test/integration/testDesignateDevice.js

```diff
@@ -43,14 +43,6 @@ describe('designateDevice.js integration tests: handleDesignateDevice', () => {
       )
     })
 
-    it('should return a JSON response body', async () => {
```

Comment:
But now that I'm not using `JSON.stringify(...)`, this test fails and I don't have a good one to replace it with. I guess I just have to make sure that I continue doing my returns with the same pattern as the existing ones.

Reply:
i don't think this test is failing because you removed `JSON.stringify()` - i think it's failing because you changed `res.send()` to `res.json()` which sets the content type of the response. somewhere inside `chai-http`, `JSON.parse()` is already being applied to the body of the response now that the content type is being set. see this thread: https://stackoverflow.com/questions/43320699/chai-response-body-is-always-empty

i would suggest replacing this test with something like `expect(this.response).to.be.json` - i think it makes sense to test that our API is returning something reasonable.

Reply:
Cool. I will do that and will update the other tests to use that too. Thanks for finding that.